### PR TITLE
setup prettier and lint staged

### PR DESCRIPTION
**Pull request by zwwuu**

### CODE REVIEW

File: .husky/pre-commit
Comment (JoeBouchard) [inline .husky/pre-commit:1]: What does Husky add to the project? I don't have much experience with the library.

**Review by JoeBouchard — COMMENTED** [review]
Looks good! It's mostly formatting stuff, so there's not much to say. My only question is about Husky. What does it bring to the project?

File: .husky/pre-commit
Comment (zwwuu) [reply .husky/pre-commit:1]: It is library for managing git hooks. In this case, I set it to format the files on pre-commit.

Comment (JoeBouchard) [reply .husky/pre-commit:1]: Sounds good!

**Review by JoeBouchard — APPROVED** [review]

**Pull request merged**